.travis.yml

```diff
@@ -20,6 +22,11 @@ install:
   - npm install
 
 before_install:
+  - wget -q -O - https://dl-ssl.google.com/linux/linux_signing_key.pub | sudo apt-key add -
+  - sudo sh -c 'echo "deb http://dl.google.com/linux/chrome/deb/ stable main" >> /etc/apt/sources.list.d/google.list'
+  - sudo apt-get update -qq
```

Comment:
It's not particularly clear why this makes sense.  I'm guessing it's because that command happens to look in the file you just appended to?  Is there a way to do this with a command-line switch rather than using an append side-effect in order to make it more readable?  If not, commentary could help here too.